modules/xmpp/JingleSession.js

```diff
@@ -4,6 +4,7 @@ var SDPDiffer = require("./SDPDiffer");
 var SDPUtil = require("./SDPUtil");
 var SDP = require("./SDP");
 var RTCBrowserType = require("../../service/RTC/RTCBrowserType");
+var RTC = require("../RTC/RTC");
```

Comment:
Good point, 
but then line 978 has to be fixed from RTC.switchVideoStreams to APP.RTC.switchVideoStreams


Am 03.02.2015 um 14:47 schrieb hristoterezov <notifications@github.com>:

> In modules/xmpp/JingleSession.js:
> 
> > @@ -4,6 +4,7 @@ var SDPDiffer = require("./SDPDiffer");
> >  var SDPUtil = require("./SDPUtil");
> >  var SDP = require("./SDP");
> >  var RTCBrowserType = require("../../service/RTC/RTCBrowserType");
> > +var RTC = require("../RTC/RTC");
> The RTC interface is accessible trough the APP object (APP.RTC). Do we need that line?
> 
> —
> Reply to this email directly or view it on GitHub.
>